bls_signature/src/test.rs

```diff
@@ -96,18 +96,17 @@ fn sign_and_aggregate(env: &Env, msg: &Bytes) -> BytesN<192> {
     bls.g2_msm(vec_msg, vec_sk).to_bytes()
 }
 
-fn create_client(e: &Env) -> IncrementContractClient<'_> {
-    IncrementContractClient::new(e, &e.register(IncrementContract {}, ()))
-}
-
 #[test]
 fn test() {
     let env = Env::default();
     let pk = aggregate_pk_bytes(&env);
     env.mock_all_auths();
 
-    let client = create_client(&env);
-    client.init(&pk);
+    let contract_id = env.register(
+        IncrementContract {},
```

Comment:
We should change the increment contract so that it is defined as `pub struct IncrementContract;` instead of as `pub struct IncrementContract {}`, then the `{}` will not be required here.